ports/unix/variants/coverage/usercmodule/cppexample/example.cpp

```diff
@@ -0,0 +1,18 @@
+extern "C" {
+    #include "py/obj.h"
+}
+
+#include <testcppinc.h>
```

Comment:
This header is empty, so maybe remove it?  Or put ` extern mp_obj_t cppfunc(mp_obj_t a_obj, mp_obj_t b_obj);` in it so that decl doesn't need to be repeated?

Reply:
This was actually just to test if passing `-I$(EXAMPLE_MOD_DIR)` via CXXFLAGS_USERMOD works, but indeed putting the declaration in it would actually put it to use so I'll do that.